### PR TITLE
Create Time Inclusion Metric

### PR DESCRIPTION
**Pull request by trajaram23**

### CODE REVIEW

File: focus-areas/time-inclusion.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Doing a presentation within a certain time block for certain time zones
+- Preparing and sending a recording to be live within an hour
+- Network bandwidth options by platform for both live participation and video streaming at a later date
+
+### Filters (optional)
+
+### Visualizations (optional)
+
+### Tools Providing the Metric (optional)
+
```
Comment (Nebrethar) [inline focus-areas/time-inclusion.md:25]: Suggestion: Remove optional headers

File: focus-areas/time-inclusion.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## Description
+
+Global accessibility is vital due to the role it brings in choosing when an event should take place. Time Inclusion for Virtual Events ensures support for participation from a global community during an event. Through this metric, we are able to be more inclusive to attendees and speakers in all regions of the world. 
+
+## Objectives
+ Global accessibility should be understood when creating a virtual event in order to allow attendees and speakers in all regions to benefit from the event. The goal of this metric is to help organizers of virtual events develop a reasonable method to ensure those in different time zones feel included.
+
+## Implementation
+
+Examples of Global Inclusion which can be implemented include:
```
Comment (Nebrethar) [inline focus-areas/time-inclusion.md:14]: This one was my bad.

Suggestion: Change "Global Inclusion" to "Time Inclusion"

**Review by Nebrethar — CHANGES_REQUESTED** [review]
Great work! I left a couple of suggestions, overall nice job.

The DCO (failed check) is complicated, so I'll leave it up to @klumb on whether they want to set it to "pass" for this PR.

**Review by Nebrethar — APPROVED** [review]

File: focus-areas/time-inclusion.md (hunk 2 above)
Comment (sgoggins) [reply focus-areas/time-inclusion.md]: Please add this text below the metric title: 

### This metric is a release candidate. To comment on this metric please see Issue [#[put the respective Issue Number here]](URL to issue). Following a comment period, this metric will be included in the next regular release.

Also, please open an corresponding issue, and link that specific issue in the placeholder above. 


There is also a TAG to add.

Comment (sgoggins) [reply focus-areas/time-inclusion.md]: "Metrics Candidate Release" is the tag to add to the issue.

**Pull request merged**